test/metabase/sync/analyze/query_results_test.clj

```diff
@@ -99,3 +104,19 @@
     (mt/with-temp Card [card {:dataset_query {:database (mt/id), :type :native, :native {:query "select longitude from venues"}}}]
       (is (= (select-keys mock.u/venue-fingerprints [:longitude])
              (name->fingerprints (query->result-metadata (query-for-card card))))))))
+
+(defn- timeseries-dataset
+  []
+  (->> {:aggregation [[:count]]
+        :breakout    [[:datetime-field [:field-id (data/id :checkins :date)] :month]]}
+       (mt/run-mbql-query checkins)
+       :data))
+
+(deftest error-resilience-test
+  (testing "Data should come back even if there is an error during fingerprinting"
+    (is (= 36 (with-redefs [fprint/earliest sync-test/crash-fn]
+                (-> (timeseries-dataset) :rows count)))))
+  (testing "Data should come back even if there is an error when calculating insights"
+    (testing "Data should come back even if there is an error during fingerprinting"
```

Comment:
This line looks like a copy-and-paste leftover, and should probably be removed, right?